pkg/controller/metrics/metrics.go

```diff
@@ -265,13 +285,55 @@ func (mc *Calculator) Start(stopCh <-chan struct{}) error {
 			}
 		}
 
+		mc.calculateSelectorSyncSetMetrics(mcLog)
+
 		elapsed := time.Since(start)
 		mcLog.WithField("elapsed", elapsed).Info("metrics calculation complete")
 	}, mc.Interval, stopCh)
 
 	return nil
 }
 
+func (mc *Calculator) calculateSelectorSyncSetMetrics(mcLog log.FieldLogger) {
+	mcLog.Debug("calculating metrics across all SyncSetInstances")
+	ssis := &hivev1.SyncSetInstanceList{}
+	err := mc.Client.List(context.Background(), ssis)
+	if err != nil {
+		mcLog.WithError(err).Error("error listing all SyncSetInstances")
+		return
+	}
+
+	sssInstancesTotal := map[string]int{}
+	sssInstancesUnappliedTotal := map[string]int{}
+
+	ssInstancesTotal := 0
+	ssInstancesUnappliedTotal := 0
+	for _, ssi := range ssis.Items {
+		if sss := ssi.Spec.SelectorSyncSet; sss != nil {
+			// Process SyncSetInstances with a SelectorSyncSet reference:
+			sssInstancesTotal[sss.Name]++
+
```

Comment:
I would like to see this empty line removed to make the if block look the same as the else block.